### DISCUSSION TITLE
Taskbar - Centered according to free space

### DISCUSSION THREAD

**Discussion by TakitIsy** [valinet/ExplorerPatcher#1656] [Ideas]
Would it be possible to center the taskbar according to the free space ?
I mean, I've got a bunch of icons in the notification area, and the taskbar tabs are really close to them.
It would be much nicer if the taskbar tabs were centered in the free space.
Here's what I mean : [Image](https://prnt.sc/VQTy71Xr3BzZ)

Thanks for the great work anyway, I love it. :-)
( Just in case: Sorry if the question has already been asked. )

**Comment by pyrates999** [comment]
Unless the code is already there, Microsoft would be the ones to implement this.  Explorer patcher only enables existing code that Microsoft has chosen to disable.
